src/main/webapp/resources/js/jquery-latest.js

```diff
@@ -0,0 +1,10308 @@
+/*!
```

Comment:
Lepiej takie rzeczy trzymać w resourcach, dlatego, ze jak Ci net padnie, to masz bibliotekę przy sobie, lub server z tym plikiem może paść i nasz projekt przy okazji również :)